rest-api/src/main/java/uk/gov/gchq/gaffer/rest/service/SimpleOperationService.java

```diff
@@ -76,6 +86,38 @@ public Object execute(final OperationChain opChain) {
         return execute(opChain, false);
     }
 
+    @SuppressFBWarnings
+    @Override
+    public ChunkedOutput<String> executeChunked(final OperationChain<CloseableIterable<Element>> opChain) {
```

Comment:
This method doesn't seem to work in swagger. I added some elements then tried the following:

```json
{
  "operations": [
    {
      "class" : "uk.gov.gchq.gaffer.operation.impl.get.GetAllElements"
    }
  ]
}
```